devops/production-template/playbook.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Deploy Girder
+  hosts: all
+  vars:
+    ansible_python_interpreter: auto
```

Comment:
What does this do? Seems like this would be the default?

Reply:
This is just a meme you need to ensure that Ansible uses Python 3 on the target. Once Ansible 2.12 drops, it will no longer be necessary. See https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html